src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -168,7 +168,7 @@ public bool CanComplete()
             int associatedServicesCount = OrderItems.Count(o => o.CatalogueItem.CatalogueItemType.Equals(CatalogueItemType.AssociatedService));
 
             var solutionAndAssociatedServices = catalogueSolutionsCount > 0
-                && associatedServicesCount > 0;
+                || associatedServicesCount > 0;
```

Comment:
This condition is checking for the scenario where both a solution and an associated service exist so it needs to be an `AND` condition. Changing this condition to an `OR` makes it possible to complete the order in scenarios where it shouldn't be possible (please see my comments below on the integration tests). Therefore this change will need to be reverted.

Reply:
Revert is done and changes in OrderController file is pushed.